src/ai/backend/manager/api/session.py

```diff
@@ -1245,6 +1246,11 @@ async def start_service(request: web.Request, params: Mapping[str, Any]) -> web.
     wsproxy_addr = sgroup["wsproxy_addr"]
     if not wsproxy_addr:
         raise ServiceUnavailable("No coordinator configured for this resource group")
+    wsproxy_status = await query_wsproxy_status(wsproxy_addr)
+    if wsproxy_status.get("advertise_address"):
+        wsproxy_advertise_addr = wsproxy_status["advertise_address"]
```

Comment:
We can avoid dupliacte lookups it using the walrus operator:
```suggestion
    if advertised_addr := wsproxy_status.get("advertise_address"):
        wsproxy_advertise_addr = advertised_addr
```